tests/virt_autotest/guest_installation_run.pm

```diff
@@ -37,6 +37,12 @@ sub get_script_run {
     my ($regcode, $regcode_ltss) = get_guest_regcode(separator => '|');
     my $kernel_params = get_guest_settings(settings => 'GUEST_EXT_KERNEL_PARAMS', separator => '|');
     $pre_test_cmd .= " -f \"" . $guest_pattern . "\" -n " . $parallel_num . " -r " . " -e \"" . $regcode . "\" -E \"" . $regcode_ltss . "\" -k \"" . $kernel_params->{'GUEST_EXT_KERNEL_PARAMS'} . "\"";
+    if ($guest_pattern =~ /sles-16/) {
```

Comment:
Any better condition than ```$guest_pattern =~ /sles-16/``` ? 

It seems that this block handles agama relevant content, so I think use ```is_agama_guest```.

Reply:
As part of the **sles-16** S390x guest installation automation process, the only addition in the OSD section is the processing of the variable $agama_mirror_repo, which is obtained using the function `get_var('MIRROR_HTTP') `and then passed to the subsequent file `/usr/share/qa/virtautolib/lib/virt-install.sh`.